pages/1.10/installing/oss/cloud/gce/index.md

```diff
@@ -6,283 +6,241 @@ navigationTitle: GCE
 
-1.  Run this command to generate the keys:
+To apply the configuration file, you can use this command below.
 
-    ```bash
-    ssh-keygen -t rsa -f ~/.ssh/id_rsa -C ajazam
-    ```
+```bash
+terraform apply -var-file desired_cluster_profile.tfvars
+```
 
-    Do not enter a password when prompted.
+#### Advance YAML Configuration
 
-1.  Make a backup copy of `id_rsa.pub`.
+In this project we have designed to be flexible. Here are the example working variables that allows very deep customization by using a single `tfvars` file.
 
-1.  Open RSA pub key:
+This file can have as little to as large as the DC/OS advance YAML configuration below.
 
-    ```bash
-    vi ~/.ssh/id_rsa.pub
-    ```
 
-    You should see something like this:
+For advance users with stringent requirements, here are the DC/OS flags examples where you can simply paste your YAML configuration in your desired_cluster_profile.tfvars. The alternative to YAML is to convert it to JSON.
 
-    ```bash
-    ssh-rsa abcdefghijklmaasnsknsdjfsdfjs;dfj;sdflkjsd ajazam
-    ```
+```bash
+$ cat desired_cluster_profile.tfvars
+dcos_version = "1.10.2"
+os = "centos_7.3"
+num_of_masters = "3"
+num_of_private_agents = "2"
+num_of_public_agents = "1"
+expiration = "6h"
+dcos_security = "permissive"
+dcos_cluster_docker_credentials_enabled =  "true"
+dcos_cluster_docker_credentials_write_to_etc = "true"
+dcos_cluster_docker_credentials_dcos_owned = "false"
+dcos_cluster_docker_registry_url = "https://index.docker.io"
+dcos_overlay_network = <<EOF
+# YAML
+    vtep_subnet: 44.128.0.0/20
+    vtep_mac_oui: 70:B3:D5:00:00:00
+    overlays:
+      - name: dcos
+        subnet: 12.0.0.0/8
+        prefix: 26
+EOF
+dcos_rexray_config = <<EOF
+# YAML
+  rexray:
+    loglevel: warn
+    modules:
+      default-admin:
+        host: tcp://127.0.0.1:61003
+    storageDrivers:
+    - ec2
+    volume:
+      unmount:
+        ignoreusedcount: true
+EOF
+dcos_cluster_docker_credentials = <<EOF
+# YAML
+  auths:
+    'https://index.docker.io/v1/':
+      auth: Ze9ja2VyY3licmljSmVFOEJrcTY2eTV1WHhnSkVuVndjVEE=
+EOF
+gcp_ssh_pub_key_file = "INSERT_PUBLIC_KEY_PATH_HERE"
+```
+_Note: The YAML comment is required for the DC/OS specific YAML settings._
 
-1.  Prefix your username, followed by a colon, to the above line. Also
+## Upgrading DC/OS
 
-    ```bash
-    ajazam:ssh-rsa abcdefghijklmaasnsknsdjfsdfjs;dfj;sdflkjsd ajazam
-    ```
+You can upgrade your DC/OS cluster with a single command. This terraform script was built to perform installs and upgrade from the inception of this project. With the upgrade procedures below, you can also have finer control on how masters or agents upgrade at a given time. This will give you the ability to change the parallelism of master or agent upgrades.
```

Comment:
In this sentence: "This terraform script was built to perform installs and upgrade from the inception of this project."
change "upgrade" to "upgrades"